python/ray/dashboard/modules/metrics/dashboards/train_dashboard_panels.py

```diff
@@ -28,7 +28,7 @@ def next():
     unit="",
     targets=[
         Target(
-            expr="sum(ray_train_controller_state{{{global_filters}}}) by (ray_train_run_name, ray_train_controller_state)",
+            expr='sum(ray_train_controller_state{{SessionName=~"$SessionName", ray_train_run_name=~"$TrainRunName", ray_train_run_id=~"$TrainRunId"}}) by (ray_train_run_name, ray_train_controller_state)',
```

Comment:
@alanwguo I made these all explicit, but is it better to have `{{global_filters}}` instead?

Reply:
you need to keep global_filters on every single panel because there are env vars that lets users add additional global filters.